src/hearings/containers/request-hearing/hearing-stage/hearing-stage.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="govuk-width-container">     
+  <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">        
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <form class="form"  [formGroup]="stageForm">
+          <span class="govuk-caption-l">Request a hearing for {{ fullName }} vs DWP</span>
```

Comment:
this should use the generic component 
    exui-hearing-parties-title
```suggestion
          <exui-hearing-parties-title></exui-hearing-parties-title>
```
refer to other components please.

Reply:
This not part off the storey

Reply:
@rayliangatos this is not part off the story

Reply:
This done by it has not marked it @rayliangatos

Reply:
we need to use a generic component to render all the titles. So you just need to change what I suggested.